libs/components/split-view/src/lib/modules/split-view/split-view-workspace-footer.component.scss

```diff
@@ -1,11 +1,32 @@
 @use 'libs/components/theme/src/lib/styles/variables' as *;
+@use 'libs/components/theme/src/lib/styles/compat-tokens-mixins' as compatMixins;
+
+@include compatMixins.sky-default-overrides(
+  '.sky-split-view-workspace-footer'
+) {
+  --sky-override-split-view-workspace-footer-padding: 10px;
+  --sky-override-split-view-workspace-footer-box-shadow: 0 -3px 3px 0
+    #{$sky-color-gray-20};
+}
+
+@include compatMixins.sky-modern-overrides('.sky-split-view-workspace-footer') {
+  --sky-override-split-view-workspace-footer-box-shadow: 0 -3px 3px 0
+    #{$sky-color-gray-20};
+}
 
 :host {
   display: block;
   flex-shrink: 0;
 }
 
 .sky-split-view-workspace-footer {
-  padding: 10px;
-  box-shadow: 0 -3px 3px 0 $sky-color-gray-20;
+  padding: var(
+    --sky-override-split-view-workspace-footer-padding,
+    var(--sky-space-inset-balanced-s)
+  );
+  // Remove this rule and use the `sky-elevation-3` class directly when only v2 modern is supported
```

Comment:
This comment says to use the class directly, but it is also added to the footer in this PR. Is that the intention? And this style just overwrites it for modern's benefit?

Reply:
That is correct. I added it so it was already there when we remove the rule.